maintainers/maintainer-list.nix

```
@@ -19574,6 +19574,12 @@
     githubId = 25278;
     name = "Otavio Salvador";
   };
+  otheanalyst = {
+    name = "oTheAnalyst";
+    email = "unfixable47@proton.me";
+    github = "oTheAnalyst";
+    githubId = "30744769";
```

must be a number, not a string
```suggestion
    githubId = 30744769;
```
please, pay attention to the CI fails, we're not gonna do it for you forever